### DISCUSSION TITLE
How enforce dark mode?

### DISCUSSION THREAD

**Discussion by aldoprogra** [directus-labs/agency-os#88] [General]
Hi! I am trying to find a way to enforce dark mode since the first loading of the landing page, I tried by modifying the nuxt config without luck. Does anyone knows ?

**Comment by Abdallah-Awwad** [comment]
You can find the file you're looking for here: 
https://github.com/directus-labs/agency-os/blob/main/components/DarkModeToggle.vue#L7 
I tested it on my end, but there seems to be an issue with the toggle button in the footer.

**Reply by aldoprogra** [reply]
hi! Thank you for the response, I tried but doesn't seem to work. On your end, after changing it, will be always dark mode until press again the toggle? even if you change the browser without cache?

**Reply by Abdallah-Awwad** [reply]
Yes, if you replaced the `'light` to `dark` in line 7, and you cleared the cache or used another browser, the default mode would be the dark mode until you toggle it.

**Reply by aldoprogra** [reply]
thank you, I messed up the ternary operator, but with your help I spotted the problem.

**Reply by aldoprogra** [reply]
No by the way still do not work on my side.
